monero/caip10.md

```diff
@@ -0,0 +1,76 @@
+---
+namespace-identifier: monero-caip10
+title: Monero - Addresses
+author: silverpill (@silverpill)
+discussions-to: https://github.com/ChainAgnostic/namespaces/issues/41
+status: Draft
+type: Informational
+created: 2023-09-01
+requires: ["CAIP-10"]
+---
+
+# CAIP-10
+
+*For context, see the [CAIP-10][] specification.*
+
+## Rationale
+
+CAIP-10 defines a pattern for identifying an account in a blockchain. 
+Monero chains use UTXO model and [Monero addresses][] are functionally similar to accounts in other namespaces.
+
+## Semantics
+
+Monero has several types of addresses: [standard address](https://monerodocs.org/public-address/standard-address/), [subaddress](https://monerodocs.org/public-address/subaddress/) and [integrated address](https://monerodocs.org/public-address/integrated-address/). 
+Any address contains network and address type identifiers, public spend key, public view key, a checksum, and may contain additional data depending on the address type. 
+[Base58-monero][base58-monero] encoding is used to encode the address into a string.
+
+CAIP-10 "accounts" should not be confused with [Monero accounts][], which are used in Monero wallets for grouping related addresses and transactions.
```

Comment:
Do you mean that in a Monero wallet, *counterparties'* various addresses are grouped into a logical unit representing the actor?  This would align with other UTXO systems and my mental model, but making it painfully explicit is usually my goal here :D

Also, does the term also organize transactions on, say, block explorers or other ledger views? 
```suggestion
CAIP-10 "accounts" should not be confused with "[Monero accounts][]", a term used in Monero wallets for grouping related addresses and transactions by actor.
```

Reply:
>Do you mean that in a Monero wallet, counterparties' various addresses are grouped into a logical unit representing the actor?

No, there are no on-chain links between addresses. Accounts only exist in the wallet, mostly for convenience. There was a similar entity in Bitcoin software, although they removed it later (see this RPC method: https://bitcoincore.org/en/doc/0.16.0/rpc/wallet/getaccount/).

Overall, Monero addresses are very similar to Bitcoin and Ethereum addresses.

>Also, does the term also organize transactions on, say, block explorers or other ledger views?

No.

Reply:
>a term used in Monero wallets for grouping related addresses and transactions by actor.

I think accounts are usually used to categorize one's own finances.

Reply:
Right, so it's a UX construct, not any kind of onchain data structure.  That makes sense-- definitely worth warning people new to Monero that they will see the term "account" used in this way and it will NOT be a 1:1 map to CAIP-10s!